people-and-planet-ai/image-classification/README.ipynb

```diff
@@ -0,0 +1,808 @@
+        "id": "r7INPuD2I2vV"
+      },
+      "source": [
+        "🕵️ While analyzing the model’s evaluation, you may notice that many species of the same family are very similar and it may be confusing the model. This means it might be worth trying to classify the image to a family instead of a specific species, or get more images from different angles/proximity to detect it better."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MGl_1UyiGVl6"
+      },
+      "source": [
+        "# 🧹 Cleaning up to avoid additional costs\n",
+        "\n",
+        "To avoid incurring charges to your Google Cloud account for the resources used in this tutorial, either delete the project that contains the resources, or keep the project and delete the individual resources.\n",
+        " \n",
+        "If you wish to delete the project you created for this tutorial (it’s the easiest way to eliminate billing), here are the steps:"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nv7phC7WGZuT"
+      },
+      "source": [
+        "## Deleting the project\n",
+        "\n",
+        "The easiest way to eliminate billing is to delete the project that you created for the tutorial.\n",
+        "\n",
+        "To delete the project:\n",
+        "\n",
+        "> ⚠️ **Caution**: Deleting a project has the following effects:\n",
+        ">\n",
+        "> - **Everything in the project is deleted.** If you used an existing project for this tutorial, when you delete it, you also delete any other work you've done in the project.\n",
+        "> - **Custom project IDs are lost.** When you created this project, you might have created a custom project ID that you want to use in the future. To preserve the URLs that use the project ID, such as an `appspot.com` URL, delete selected resources inside the project instead of deleting the whole project.\n",
+        ">\n",
+        "> If you plan to explore multiple tutorials and quickstarts, reusing projects can help you avoid exceeding project quota limits.\n",
+        "\n",
+        "1. In the Cloud Console, go to the **Manage resources** page.\n",
+        "\n",
+        "  <button>\n",
+        "\n",
+        "  [Go to Manage resources](https://console.cloud.google.com/iam-admin/projects)\n",
+        "\n",
+        "  </button>\n",
+        "\n",
+        "1. In the project list, select the project that you want to delete, and then click **Delete**.\n",
+        "\n",
+        "1. In the dialog, type the project ID, and then click **Shut down** to delete the project.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Em4E_RNv2BZM"
+      },
+      "source": [
+        "## 🎉 Congratulations, you have successfully completed this code lab!\n",
+        " \n",
+        "🙌 You made it this far, congrats.\n",
+        "Next step from here on, is you can go deeper by visiting the\n",
```

Comment:
Phrased a little awkwardly, perhaps "Dive deeper by visiting the code on GitHub."

Reply:
Thanks, rephrased